### PR TITLE
Add gp42 repository

### PR DESCRIPTION
**Pull request by gp42**
Following the conversation in: https://github.com/helm/charts/pull/17398, I would like to publish the `calico-aws` chart on https://hub.helm.sh.

This is a new Helm chart for running Project Calico on AWS EKS.

Signed-off-by: Gennady Potapov <gennadiy.potapov@gmail.com>

### CODE REVIEW

**Comment by cpanato** [comment]
@gp42 please sign the DCO

**Comment by gp42** [comment]
:/ strange `git log` shows me:
```
commit 74ec6dceaf4152abcb213a813ef3de7491531eaa (HEAD -> add_calico-aws, origin/add_calico-aws)
Author: Gennady Potapov <gennadiy.potapov@gmail.com>
Date:   Wed Dec 11 11:43:29 2019 +0200

    sign

    Signed-off-by: Gennady Potapov <gennadiy.potapov@gmail.com>
```

File: repos.yaml [hunk 1]
```diff
@@ -683,6 +683,11 @@ repositories:
     maintainers:
       - email: shih5@chinaunicom.cn
         name: Hao Shi
+  - name: gp42
+    url: https://ops42.org
```
Comment (mattfarina) [inline repos.yaml]: The URL in both files should be the same and to the charts repository. Can you update this to be `https://ops42.org/helm-charts`.

**Comment by mattfarina** [comment]
@gp42 Thanks for the PR and the addition. Just one little change needs to be made.

File: repos.yaml (hunk 1 above)
Comment (gp42) [reply repos.yaml]: updated

**Pull request merged**